src/javascript/app/pages/user/account/settings/personal_details.js

```diff
@@ -228,8 +228,10 @@ const PersonalDetails = (() => {
                     return;
                 }
                 const get_settings    = data.get_settings;
-                const has_required_mt = get_settings.tax_residence && get_settings.tax_identification_number &&
-                    get_settings.citizen;
+                const has_required_mt = (Client.get('landing_company_shortcode') === 'costarica' ?
```

Comment:
isn't it better to check with the new field `landing_company.config.tax_details_required` instead of CR?

Reply:
sure, it it only for mt accounts?

Reply:
that's my understanding. you can check with the QA team